components/shell/MobileMenu.tsx

```diff
@@ -119,9 +116,6 @@ export function MobileMenu({ toggleTheme }: { toggleTheme: () => void }) {
                   <nav className="flex h-full flex-col">
                     <div className="space-y-1">
                       {NAV_ITEMS.map((item) => {
-                        if (item.serviceAccountOnly && !isServiceAccount) {
```

Comment:
This check will just need to move to the inside of the page so that the actual config editor is only shown to service accounts.